{"text": "2024-09-05 06:26:05 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:02.014787", "seconds": 2.014787}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-games/epic-games/epic/src/claim.py"}, "function": "stash", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3598, "name": "MainProcess"}, "thread": {"id": 140522279144320, "name": "MainThread"}, "time": {"repr": "2024-09-05 06:26:05.602130+08:00", "timestamp": 1725488765.60213}}}
{"text": "2024-09-05 06:26:13 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:10.318806", "seconds": 10.318806}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-games/epic-games/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3598, "name": "MainProcess"}, "thread": {"id": 140522279144320, "name": "MainThread"}, "time": {"repr": "2024-09-05 06:26:13.906149+08:00", "timestamp": 1725488773.906149}}}
{"text": "2024-09-05 06:26:44 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:00:40.563206", "seconds": 40.563206}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-games/epic-games/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3598, "name": "MainProcess"}, "thread": {"id": 140522279144320, "name": "MainThread"}, "time": {"repr": "2024-09-05 06:26:44.150549+08:00", "timestamp": 1725488804.150549}}}
{"text": "2024-09-05 06:27:14 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:01:10.565311", "seconds": 70.565311}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-games/epic-games/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3598, "name": "MainProcess"}, "thread": {"id": 140522279144320, "name": "MainThread"}, "time": {"repr": "2024-09-05 06:27:14.152654+08:00", "timestamp": 1725488834.152654}}}
{"text": "2024-09-05 06:27:44 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:01:40.566717", "seconds": 100.566717}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-games/epic-games/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3598, "name": "MainProcess"}, "thread": {"id": 140522279144320, "name": "MainThread"}, "time": {"repr": "2024-09-05 06:27:44.154060+08:00", "timestamp": 1725488864.15406}}}
{"text": "2024-09-05 06:27:44 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3598), thread 'MainThread' (140522279144320): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-games/epic-games/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic-games/epic-games/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic-games/epic-games/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic-games/epic-games/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic-games/epic-games/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:01:40.683737", "seconds": 100.683737}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-games/epic-games/epic/src/claim.py"}, "function": "run", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3598), thread 'MainThread' (140522279144320):", "module": "claim", "name": "__main__", "process": {"id": 3598, "name": "MainProcess"}, "thread": {"id": 140522279144320, "name": "MainThread"}, "time": {"repr": "2024-09-05 06:27:44.271080+08:00", "timestamp": 1725488864.27108}}}
